### DISCUSSION TITLE
Nodemcu + lora sx1262

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#289] [General]
Hi, 
I have problem with nodemcu and RadioLib library, I uploaded example "SX126x_Receive" and setup correct pins, after that nodemcu keeps reseting after while in loop with console output:
`[SX1262] Initializing ... H⸮⸮⸮!⸮⸮⸮⸮`

Correct pin settings:  NSS pin:   8, DIO1 pin:  3, NRST pin:  2, BUSY pin:  4.

With default (wrong) pins, serial console output looks like this:
`H!⸮⸮!⸮⸮⸮⸮[SX1262] Initializing ... failed, code -705
18:23:49.766 -> 
18:23:49.766 -> --------------- CUT HERE FOR EXCEPTION DECODER ---------------
18:23:49.813 -> 
18:23:49.813 -> Soft WDT reset
18:23:49.813 -> 
18:23:49.813 -> >>>stack>>>
18:23:49.861 -> 
18:23:49.861 -> ctx: cont
18:23:49.861 -> sp: 3ffffde0 end: 3fffffc0 offset: 01a0
18:23:49.909 -> 3fffff80:  fffffd3f 0000000a 3ffee364 402010a1  
18:23:49.956 -> 3fffff90:  0000000a 00000008 3fcccccd 00000000  
18:23:50.001 -> 3fffffa0:  3fffdad0 00000000 3ffee38c 40204660  
18:23:50.047 -> 3fffffb0:  feefeffe feefeffe 3ffe84e0 40100ff9  
18:23:50.094 -> <<<stack<<<
18:23:50.141 -> 
18:23:50.141 -> --------------- CUT HERE FOR EXCEPTION DECODER ---------------`

It seems that, it's reseting at radio.begin() function. Can you help me with this problem?

**Comment by frogg1ven** [comment]
problem solver, pins were wrong, nodemcu has separate pin numbers

**Reply by jgromes** [reply]
The pins provided in the examples are just that - examples, you can set them to pretty much anything you want. Of course it has match the hardware ;)
